core/app/services/spree/cart/estimate_shipping_rates.rb

```diff
@@ -0,0 +1,28 @@
+module Spree
+  module Cart
+    class EstimateShippingRates
+      prepend Spree::ServiceModule::Base
+
+      def call(order:, country_iso: nil)
+        country_id = if country_iso.present?
+                       ::Spree::Country.by_iso(country_iso).id
```

Comment:
@jkojro also if there isn't a country with this ISO / ISO3 this will return a nil error